pkg/skaffold/test/custom/custom.go

```diff
@@ -190,6 +193,14 @@ func (ct *Runner) getEnv(imageTag string) ([]string, error) {
 	}
 
 	envs = append(envs, util.OSEnviron()...)
+
+	// add minikube docker env vars to command env context if minikube cluster detected
+	localDaemon, err := docker.NewAPIClient(ct.cfg)
+	if err != nil {
+		return nil, fmt.Errorf("getting docker client information: %w", err)
```

Comment:
Lets make this an actionable error message.
See here to add a new distinct error code and add it here https://github.com/GoogleContainerTools/skaffold/blob/master/DEVELOPMENT.md#adding-actionable-error-messages-to-code

Depending on the error message, if `minikube docker-env` returns error, the actionable messages could be
1) If minikube is stopped and not running.  You can try reuse the error code [`CHECK_MINIKUBE_STATUS `](https://github.com/GoogleContainerTools/skaffold/blob/master/proto/enums/enums.proto#L454)
2) docker is stopped and not running.

Reply:
I am ok doing this in a follow up PR

Reply:
Will do in a followup PR, tracking the work in #5820